utilities/policy_utilities.mak

```diff
@@ -116,9 +119,9 @@ $(O)/key_utility.o: $(US)/key_utility.cc $(INC_DIR)/support.h $(INC_DIR)/certifi
 	@echo "compiling key_utility.cc"
 	$(CC) $(CFLAGS) -c -o $(O)/key_utility.o $(US)/key_utility.cc
 
-#$(US)/certifier.pb.cc $(I)/certifier.pb.h: $(CERT_SRC)/certifier.proto
-#	$(PROTO) -I$(S) --cpp_out=$(US) $(CERT_SRC)/certifier.proto
-#	mv certifier.pb.h $(I)
+$(CERT_SRC)/certifier.pb.cc $(I)/certifier.pb.h: $(CERT_SRC)/certifier.proto
+	$(PROTO) -I$(S) --proto_path=$(CERT_SRC) --cpp_out=$(CERT_SRC) $(CERT_SRC)/certifier.proto
```

Comment:
The real-fix seems to be the use of `--proto_path=$(CERT_SRC)`. I found this in `man git`.

The change on L122 cleans-up some dependency listing ... which seem ok to me.

Again, I am no Make-expert ... so another pair of eyes during review will be good!
All this mess seems to work fine in manual & CI testing.

Reply:
Thanks for fixing this. We might want to do something similar in cert_utility.mak too so that we never duplicate the protobuf generation.

Reply:
I really don't know what this duplication is all about.

Both `protoc` invocations are emitting outputs to diff sub-dirs, to satisfy other build rules.

I'd like to not touch the other .mak file at this time. Let this go in .. and we can re-check with John once he gets to use this and test it out in his own env.